package.json

```diff
@@ -31,6 +31,7 @@
     "eslint": "^2.9.0",
     "@ljharb/eslint-config": "^4.0.0",
     "parallelshell": "^2.0.0",
+    "iconv": "^2.2.0",
```

Comment:
I totally understanding adding the exact test for your use case :-) and i'm all for that. Surely there's an easier way tho?